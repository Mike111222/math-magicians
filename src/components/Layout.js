import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import { Link } from 'react-router-dom';

const Layout = () => (
  <nav className="navBar">
    <h1 className="navHeading">Math Magicians</h1>
    <ul className="navLinks">
      <li><Link to="/"><span>Home</span></Link></li>
      <li><Link to="/calculator"><span>Calculator</span></Link></li>
      <li><Link to="/quote"><span>Quote</span></Link></li>
    </ul>
  </nav>
);

export default Layout;
