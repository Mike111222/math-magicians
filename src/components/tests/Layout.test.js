import React from 'react';
import { render } from '@testing-library/react';
import { HashRouter } from 'react-router-dom';
import Layout from '../Layout';

it('render correctly and mathes the snapshot', () => {
  const { asFragment } = render(
    <HashRouter>
      {' '}
      <Layout />
    </HashRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
