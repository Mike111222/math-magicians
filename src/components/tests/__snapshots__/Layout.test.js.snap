// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly and mathes the snapshot 1`] = `
<DocumentFragment>
   
  <nav
    class="navBar"
  >
    <h1
      class="navHeading"
    >
      Math Magicians
    </h1>
    <ul
      class="navLinks"
    >
      <li>
        <a
          href="#/"
        >
          <span>
            Home
          </span>
        </a>
      </li>
      <li>
        <a
          href="#/calculator"
        >
          <span>
            Calculator
          </span>
        </a>
      </li>
      <li>
        <a
          href="#/quote"
        >
          <span>
            Quote
          </span>
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
